storage/remote/queue_manager.go

```diff
@@ -92,15 +93,22 @@ func NewStorageQueueManager(tsdb StorageClient, queueCapacity int) *StorageQueue
 		sendLatency: prometheus.NewSummary(prometheus.SummaryOpts{
 			Namespace:   namespace,
 			Subsystem:   subsystem,
-			Name:        "sent_latency_milliseconds",
+			Name:        "send_latency_milliseconds",
 			Help:        "Latency quantiles for sending sample batches to the remote storage.",
 			ConstLabels: constLabels,
 		}),
-		sendErrors: prometheus.NewCounter(prometheus.CounterOpts{
+		failedBatches: prometheus.NewCounter(prometheus.CounterOpts{
 			Namespace:   namespace,
 			Subsystem:   subsystem,
-			Name:        "sent_errors_total",
-			Help:        "Total number of errors sending sample batches to the remote storage.",
+			Name:        "failed_batches_total",
+			Help:        "Total number of sample batches that encountered an error while sending to the remote storage.",
```

Comment:
I was thinking from the perspective of the storage queue at the end of the sentence again, but I guess that makes more sense :)